src/Options/Options.php

```diff
@@ -0,0 +1,59 @@
+<?php
+declare(strict_types=1);
+
+namespace RocketCDN\Options;
+
+/**
+ * Manages single site options using the WordPress options API.
+ */
+class Options extends AbstractOptions {
```

Comment:
```suggestion
class Options extends AbstractOptions implements OptionsInterface {
```

Reply:
The abstract class already implements the interface, do you think it's necessary to add it again here?